src/ucs/type/float8.h

```diff
@@ -236,6 +236,21 @@ ucs_fp8_unpack(ucs_fp8_t value, uint64_t min, uint64_t max)
     _UCS_FP8_IDENTIFIER(_name, _unpack)(_value)
 
 
+/**
+ * Pack a double-precision floating-point number of a given type to a single
+ * byte and then unpack it. Can be useful to comparison value with another one
```

Comment:
```suggestion
 * byte and then unpack it. Can be useful for comparison value with another one
```

Reply:
maybe: Can be useful for simulating the loss of precision when comparing with an unpacked value.